src/components/DailyGrowth/DailyGrowth.jsx

```diff
@@ -1,96 +1,136 @@
+  Tooltip,
+  XAxis,
+  YAxis
+} from "recharts";
+import Loader from "./Loader";
+import { sum } from "../../helpers/misc";
+import { Label3 } from "baseui/typography";
+import { Block } from "baseui/block";
+import moment from "moment";
 
-const CASES_KEY = 'cases';
-const DEATHS_KEY = 'deaths';
-const CURES_KEY = 'cures';
+const CASES_KEY = "cases";
+const DEATHS_KEY = "deaths";
+const CURES_KEY = "cures";
 
 function accumulateData(data) {
   let cumulativeCases = 0;
   let cumulativeDeaths = 0;
   let cumulativeCures = 0;
 
-  return Object
-    .entries(groupBy(data, 'date'))
-    .map(([date, dataPerDate]) => {
-      const casesDateCount = sum(dataPerDate.filter(el => el.key === CASES_KEY))
-      const deathsDateCount = sum(dataPerDate.filter(el => el.key === DEATHS_KEY))
-      const curesDateCount = sum(dataPerDate.filter(el => el.key === CURES_KEY))
+  return Object.entries(groupBy(data, "date")).map(([date, dataPerDate]) => {
+    const casesDateCount = sum(dataPerDate.filter(el => el.key === CASES_KEY));
+    const deathsDateCount = sum(
+      dataPerDate.filter(el => el.key === DEATHS_KEY)
+    );
+    const curesDateCount = sum(dataPerDate.filter(el => el.key === CURES_KEY));
 
-      cumulativeCases += casesDateCount;
-      cumulativeDeaths += deathsDateCount;
-      cumulativeCures += curesDateCount;
+    cumulativeCases += casesDateCount;
+    cumulativeDeaths += deathsDateCount;
+    cumulativeCures += curesDateCount;
 
-      return {
-        date: moment(date).format("DD/MM"),
-        [CASES_KEY]: cumulativeCases,
-        [DEATHS_KEY]: cumulativeDeaths,
-        [CURES_KEY]: cumulativeCures
-      };
-    });
+    return {
+      date: moment(date).format("DD/MM"),
+      [CASES_KEY]: cumulativeCases,
+      [DEATHS_KEY]: cumulativeDeaths,
+      [CURES_KEY]: cumulativeCures
+    };
+  });
 }
 
 export default function DailyGrowth() {
   const { cases, deaths, cures, isLoading } = useData();
 
+  const { t } = useTranslation();
   const [groupedData, setGroupedData] = useState(null);
   const [, theme] = useStyletron();
 
   useEffect(() => {
-    const preparedCases = cases ? cases.map(el => ({ ...el, key: CASES_KEY })) : [];
-    const preparedDeaths = deaths ? deaths.map(el => ({ ...el, key: DEATHS_KEY })) : [];
-    const preparedCures = cures ? cures.map(el => ({ ...el, key: CURES_KEY })) : [];
+    const preparedCases = cases
+      ? cases.map(el => ({ ...el, key: CASES_KEY }))
+      : [];
+    const preparedDeaths = deaths
+      ? deaths.map(el => ({ ...el, key: DEATHS_KEY }))
+      : [];
+    const preparedCures = cures
+      ? cures.map(el => ({ ...el, key: CURES_KEY }))
+      : [];
 
-    setGroupedData(accumulateData([...preparedCases, ...preparedDeaths, ...preparedCures]));
+    setGroupedData(
+      accumulateData([...preparedCases, ...preparedDeaths, ...preparedCures])
+    );
   }, [cases, deaths, cures]);
 
   return (
     <StyledCard
       style={$theme => ({
+        width: "320px",
```

Comment:
By changing this part of the app, you will actually revert this PR https://github.com/konradkalemba/korona.ws/pull/41